@vates/parse-duration/src/index.ts

```diff
@@ -2,11 +2,11 @@
 
 const ms = require('ms')
 
-exports.parseDuration = value => {
+exports.parseDuration = (value: number | string): number => {
```

Comment:
`parseDuration` will throw for other types, should we accept them in the types or not?

I don't know the best practice for this.

Reply:
We should only accept types that can be used by the function (and avoid "any").
If at runtime we should have data of an unintended type, this is handled by error handling.